### ISSUE TITLE
👌 IMPROVE: convert provided structure into primitive

### ISSUE THREAD

**Issue by mbercx** [aiidalab/aiidalab-qe#198]
**Is your feature request related to a problem? Please describe.**

Currently the `QeAppWorkChain` simply runs the structure that the user provides, without attempting to primitivize the structure. This is inefficient, since the primitive structure can be significantly smaller. Although one can think of reasons to run the conventional unit cell, these are typically not required for the layman user that simply wants to use the QEapp to calculate the band structure/PDOS.

**Describe the solution you'd like**

I think it's sensible to immediately primitivize the structure using SeeK-path and run all calculations for this structure. We should at least raise a warning that the work chain will run the primitive structure by default, and perhaps allow a flag in the advanced settings to disable this feature. I think currently the `PwBandsWorkChain` cannot be run without SeeK-path, however, so this might require some more fundamental changes.

A couple of potential issues:

* I think the `PwBandsWorkChain` will _always_ run SeeK-path, and I'm not sure if running SeeK-path twice on a structure is idempotent?
* Even though the user might be warned that the structure is primitivized, they might be surprised that the structure in the results is not the same as the one they provided. Reconstructing the original unit cell based on the relaxed primitive one is not trivial.

**Comment by yakutovicha** [comment]
I suggest addressing this by adding a structure editor that is capable of "primitivising" the structure.

This way the user has control over the process, and decide themselves on what to do:
<img width="994" alt="Screenshot 2022-02-10 at 11 43 15" src="https://user-images.githubusercontent.com/13118485/153390884-74e2fd40-d7a6-4ee8-a78c-3a6ae218e73b.png">

The way to incorporate it in the following:
```python
widget = StructureManagerWidget(
    importers=[
        StructureUploadWidget(title="From computer"),
        CodQueryWidget(title="COD"),
        StructureBrowserWidget(title="AiiDA database"),
        SmilesWidget(title="SMILES"),
    editors = [
        BasicStructureEditor(title="Basic Editor"),
        HERE, <----- Place your editor.
    ],
)
```

**Comment by unkcpz** [comment]
Thanks for @yakutovicha has implemented the structure editor widget https://github.com/aiidalab/aiidalab-qe/pull/216. Based on that, add a feature to convert to primitive cell.

**Comment by unkcpz** [comment]
Closed by #216
